libs/gi/page-team/src/TeamSetting/index.tsx

```diff
@@ -191,6 +250,19 @@ function TeamEditor({
           character" buffs, and cannot be empty.
         </Trans>
       </Alert>
+      <Grid container sx={{ justifyContent: 'space-between' }} spacing={2}>
+        <Grid item>
+          <Typography variant="h5">Selected Team Members</Typography>
```

Comment:
Opps, forgot to remind you to i10n these newly added text. I think In this line, "Team Members" would be more appropriate, and for the button, "Quick setup" can be used.
![image](https://github.com/user-attachments/assets/016ec445-a27f-4868-ae99-44faee32a080)

Reply:
perhaps we can ~~steal~~ borrow the i10n from datamine for "Quick Setup"
![image](https://github.com/user-attachments/assets/416fe0bd-2e0f-4ca9-8f6e-76aa5dfee3ae)